src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -10,8 +10,8 @@ import {
 } from 'react-native';
 import MemoDomainProfilePicture from '../../../assets/icon/DomainProfilePictureEmptyState';
 import {COLORS} from '../../../utils/theme';
-import {CircleGradient} from '../../../components/Karma/CircleGradient';
-import {fonts, normalize} from '../../../utils/fonts';
+import {fonts} from '../../../utils/fonts';
+import ProfilePicture from '../../ProfileScreen/elements/ProfilePicture';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ProfilePicture` component is a good replacement for the `CircleGradient` component. This change aligns with the PR's goal of modifying Karma score representation.

```diff
- import {CircleGradient} from '../../../components/Karma/CircleGradient';
- import {fonts, normalize} from '../../../utils/fonts';
+ import {fonts} from '../../../utils/fonts';
+ import ProfilePicture from '../../ProfileScreen/elements/ProfilePicture';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->